pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BuildConditionResponderTest.java

```diff
@@ -242,4 +242,21 @@ public void changedAndNotSuccess() throws Exception {
         j.assertLogNotContains("I FAILED", b3);
     }
 
+    @Issue("JENKINS-44993")
+    @Test
+    public void failureInPostBlock() throws Exception {
+        expect(Result.FAILURE, "failureInPostBlock")
+                .logContains("[Pipeline] { (foo)",
+                        "hello",
+                        "[Pipeline] { (" + SyntheticStageNames.postBuild() + ")",
+                        "I FAILED",
+                        "Error when executing failure post condition",
+                        "Error when executing always post condition",
```

Comment:
I would expect this line to occur before "I FAILED".

Reply:
This isn't ordered - it's just "are each of these strings present in the log somewhere?"

Reply:
@abayer 
Ah, okay.  Well, we'll need to add a `logContainsInOrder` or equivalent.
The order of these lines matters and depending on implementation they might occur in a different order, including the one you shown here is not.

Reply:
Oh, looks like you already have it implemented: 
[inLogInOrder](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java#L390)

Reply:
I completely forgot that!

Reply:
@abayer Maybe rename it to `logContainsInOrder` so it is easier to find/remember?  `inLog...` is quite odd.